### ISSUE TITLE
Carolyn Book Investigator Information Issue

### ISSUE THREAD

**Issue by msbeck** [Kamalisk/arkhamdb#249]
https://arkhamdb.com/card/98010

Carolyn's book card page says the deckbuilding requirements include Hypnotic Therapy and Rational Thought but her book cards are Foolishness: Foolish Cat of Ulthar and To Fight the Black Wind

**Comment by zzorba** [comment]
Yeah, this is actually a quirk with how FFG chose to print/template these alt-art cards.

The alt-art investigators still reference the real signature cards, but those signature cards have the keyword 'Replacement' which has an extra block of rules about how they can take the place of a characters normal signature (if they come as a pair).

**Comment by msbeck** [comment]
Ah, no worries. I didn't have another example on arkhamdb to compare. Is there a reason only a few book IDs show up? I didn't see Jenny or Roland.

I guess if the text is the same they aren't necessary for deck building but often times users come to the xyzDB sites to identify the existence of cards/card sets separate from building with them.
